coverage.txt

```diff
@@ -1,68 +1,361 @@
-
-mode: atomic
-github.com/giantswarm/gsctl/config/config.go:62.13,74.2 9 1
-github.com/giantswarm/gsctl/config/config.go:76.26,77.16 1 2
-github.com/giantswarm/gsctl/config/config.go:77.16,78.21 1 0
-github.com/giantswarm/gsctl/config/config.go:83.26,91.20 4 0
-github.com/giantswarm/gsctl/config/config.go:95.2,96.21 2 0
-github.com/giantswarm/gsctl/config/config.go:100.2,100.12 1 0
-github.com/giantswarm/gsctl/config/config.go:91.20,93.3 1 0
-github.com/giantswarm/gsctl/config/config.go:96.21,98.3 1 0
-github.com/giantswarm/gsctl/config/config.go:104.59,108.20 3 1
-github.com/giantswarm/gsctl/config/config.go:117.2,118.20 2 1
-github.com/giantswarm/gsctl/config/config.go:122.2,122.22 1 1
-github.com/giantswarm/gsctl/config/config.go:108.20,109.29 1 0
-github.com/giantswarm/gsctl/config/config.go:114.3,114.22 1 0
-github.com/giantswarm/gsctl/config/config.go:109.29,113.4 1 0
-github.com/giantswarm/gsctl/config/config.go:118.20,120.3 1 0
-github.com/giantswarm/gsctl/config/config.go:126.50,129.25 2 1
-github.com/giantswarm/gsctl/config/config.go:145.2,147.45 2 1
-github.com/giantswarm/gsctl/config/config.go:153.2,153.12 1 0
-github.com/giantswarm/gsctl/config/config.go:129.25,135.32 3 0
-github.com/giantswarm/gsctl/config/config.go:140.3,140.13 1 0
-github.com/giantswarm/gsctl/config/config.go:135.32,136.45 1 0
-github.com/giantswarm/gsctl/config/config.go:136.45,138.5 1 0
-github.com/giantswarm/gsctl/config/config.go:147.45,150.3 1 1
-github.com/giantswarm/gsctl/config/config.go:163.117,165.26 1 0
-github.com/giantswarm/gsctl/config/config.go:169.2,169.24 1 0
-github.com/giantswarm/gsctl/config/config.go:172.2,175.16 4 0
-github.com/giantswarm/gsctl/config/config.go:178.2,178.38 1 0
-github.com/giantswarm/gsctl/config/config.go:196.2,196.48 1 0
-github.com/giantswarm/gsctl/config/config.go:165.26,167.3 1 0
-github.com/giantswarm/gsctl/config/config.go:169.24,171.3 1 0
-github.com/giantswarm/gsctl/config/config.go:175.16,177.3 1 0
-github.com/giantswarm/gsctl/config/config.go:178.38,179.33 1 0
-github.com/giantswarm/gsctl/config/config.go:179.33,181.46 2 0
-github.com/giantswarm/gsctl/config/config.go:190.4,190.28 1 0
-github.com/giantswarm/gsctl/config/config.go:193.4,193.18 1 0
-github.com/giantswarm/gsctl/config/config.go:181.46,183.19 2 0
-github.com/giantswarm/gsctl/config/config.go:186.5,186.60 1 0
-github.com/giantswarm/gsctl/config/config.go:183.19,185.6 1 0
-github.com/giantswarm/gsctl/config/config.go:186.60,188.6 1 0
-github.com/giantswarm/gsctl/config/config.go:190.28,192.5 1 0
 mode: atomic
```

Comment:
This is necessary for code coverage reports. Codecov.io generates a before/after comparison for a PR based on the change of this. It's generated using `./test_coverage.sh`.